src/Symfony/Component/Console/Tester/TesterTrait.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tester;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\StreamOutput;
+
+/**
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ *
+ * @internal
+ */
+trait TesterTrait
+{
+    /** @var StreamOutput */
```

Comment:
not useful and differs with `getOutput()` return type, I would drop it

Reply:
To me it is useful actually (for autocompletion and STA). This trait needs a `StreamOutput` instance here, not any `OutputInterface`. That's why I suggested this typehint in https://github.com/symfony/symfony/pull/24819#discussion_r148958291 :)